charts/camunda-platform-alpha/templates/zeebe/1/2/3/4/5/6/7/8/z_compatibility_helpers.tpl

```diff
@@ -66,7 +66,18 @@ Zeebe Gateway.
     {{- $_ := set .Values.zeebeGateway.ingress "grpc" (deepCopy $zgIngress | mergeOverwrite .Values.zeebeGateway.ingress.grpc) -}}
 {{- end -}}
 
+{{/*
+OpenShift
+*/}}
+{{- if eq .Values.global.compatibility.openshift.adaptSecurityContext "force" -}}
+    {{- $_ := set .Values.elasticsearch.sysctlImage "enabled" false -}}
+{{- end -}}
 
+{{- if eq .Values.global.compatibility.openshift.adaptSecurityContext "force" -}}
```

Comment:
Same, we need to add a comment here about what that is and why it was added.

Reply:
comment added